deploy/runtime/cashflow.lk

```diff
@@ -241,7 +242,13 @@ function cf_om_expenses()
 	if ( is_fuel_tech() ) { cfline('cf_om_fuel_expense', 0); }
 	if ( tech == 'Biopower' ) { cfline( 'cf_om_opt_fuel_1_expense,cf_om_opt_fuel_2_expense', 0); }
 	if ( tech == 'Geothermal' ) { if (value('system_use_recapitalization')==1) { cfline ('cf_recapitalization', 0); } }
-	if ( is_fom_fin() ) { cfline('cf_utility_bill', 0 ); }
+	if ( is_fom_fin() )
+	{
+		if ( fin != 'Host Developer' )
```

Comment:
Host developer is BTM, so this line will always be true:

![image](https://user-images.githubusercontent.com/5530592/169103444-01ff14a1-4341-42e2-9754-e26670cceb58.png)

Reply:
Right. That's a redundant check, I'll remove it.